### DISCUSSION TITLE
Deno ext breaks working on node projects, help please

### DISCUSSION THREAD

**Discussion by MirKml** [denoland/vscode_deno#1042] [General]
Hello,
I read the doc a tried to setup vscode for one deno project, directory layout is
```
maindir
    nodejs-project
    deno-project
````

I installed the extension, loaded `deno-project` dir, setuped `deno.enable=true` for this workspace, save into `deno-project/.vscode`. It works fine in this workspace.

Now, I open new folder `nodejs-project` with node.js project, there is no .vscode folder inside. And Language Server for built in Typescript doesn't work anymore, it constantly writes notice

`The JS/TS language service immediately crashed 5 times. The service will not be restarted. This may be caused by a plugin contributed by one of these extensions: denoland.vscode-deno. Please try disabling these extensions before filing an issue against VS Code.`

![image](https://github.com/denoland/vscode_deno/assets/21239658/66375a92-1f13-4d14-8554-55f1052d4c90)

I checked that deno.enable isn't set for global user settings - it isn't. I checked other settings, tried again, nothing helps.
Only disabling whole deno extension works, but this isn't definitively desired.

How can I setup VS code for working on different projects - some with nodejs, some with deno?
I read docs again, found nothing what I make wrong.
I must give up deno completely.

Help me please. Deno seems very good, but it breaks me work on other pure javascript/typescript projects completely.

**Comment by bartlomieju** [comment]
@nayeemrmn please take a look - I think this might be related to the recent changes in TS server plugin.

**Comment by nayeemrmn** [comment]
Fixed in 3.33.2

**Reply by MirKml** [reply]
Thanks for such a quick fix.
